package.json

```diff
@@ -12,7 +12,7 @@
         "start": "electron .",
         "test": "vue-cli-service test:unit",
         "dev": "vue-cli-service serve ",
-        "build": "npm run build:web",
+        "build": "npm run electron-rebuild && npm run build:web",
```

Comment:
@TriPhan96 that breaks releasing wallet locally

Reply:
Ok @bassemmagdy. I will change it to:

Reply:
Hi @bassemmagdy We have updated to the code applied @tri-vo-fds recommendation. Please have a look at it. Thanks

Reply:
@tri-vo-fds @TriPhan96 still breaking because of `npm run electron-rebuild` statement

Reply:
@bassemmagdy Could you please screenshot the error you are encountered when running the release command on your machine? Maybe the node version you are using would help us to reproduce your issue with this PR

Reply:
@bassemmagdy We have update the node-hid package to make the release wallet locally, please test it again, thank you.

Reply:
@TriPhan96 it worked for releasing wallet locally only for mac, but for windows and linux I always get this [output](https://drive.google.com/file/d/1UhXbaK5SxtmhWl2sManyUDP4jbX21tH3/view?usp=sharing), I'm using Osx with node v12.18.3. Also, there are some conflicts that need to be resolved.